packages/epo-widget-lib/src/lib/utils.ts

```diff
@@ -61,6 +64,14 @@ export const getLinearScale = (
       t = Math.min(Math.max(t, 0), 1);
     }
 
-    return t * (range[1] - range[0]) + range[0];
+    return Number(
```

Comment:
add default rounding so browsers don't have to try and render to the zillionth pixel fraction